### DISCUSSION TITLE
enabling of aws paths completion

### DISCUSSION THREAD

**Discussion by aggonis** [zimfw/zimfw#544] [Q&A]
is there a setting/awy i could tell the `completion` module to use the aws paths, something like:
```
complete -C $(which aws_completer) aws
```

**Comment by antoineco** [comment]
You could add the commands described at [Command completion](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-completion.html#cli-command-completion-enable) to you `.zshrc` file, after the part which loads Zim:

```zsh
# .zshrc

# ...

source ${ZIM_HOME}/init.zsh

autoload bashcompinit && bashcompinit
complete -C "$(which aws_completer)" aws
```

Note that I omitted loading `compinit` in the snippet above, because it is already loaded by Zim's completion module.
